test/server/cards/06-WoE/SpaceInvaders.spec.js

```diff
@@ -43,7 +43,6 @@ describe('Space Invaders', function () {
             this.player1.clickCard(this.zorg);
             this.player1.clickCard(this.pelf);
             this.player1.clickPrompt('Done');
-            this.player1.clickPrompt('Right');
             expect(this.player1.player.creaturesInPlay.length).toBe(2);
             expect(this.zorg.location).toBe('play area');
             expect(this.zorg.name).toBe('Grumpus');
```

Comment:
The test fails here because the creatures have their real names, not token names, and I'm not why they aren't getting tokenized...

Reply:
I will check Space Invaders... on it

Reply:
@strib, hopefully https://github.com/keyteki/keyteki/pull/3180 solves the Space Invader's problem (and other cards as well)

Reply:
For some reason the sequential action kept getting its properties set to default in between its function calls, so we lost the `cardLocation`.  Passing it around explicitly fixed the problem.  Ready to review!